hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientLockProxy.java

```diff
@@ -87,7 +87,7 @@ public void lock(long leaseTime, TimeUnit timeUnit) {
     }
 
     public void forceUnlock() {
-        ClientMessage request = LockForceUnlockParameters.encode(getName());
+        ClientMessage request = LockForceUnlockParameters.encode(getName(), ThreadUtil.getThreadId());
```

Comment:
I removed this threadId intentionally before. Is it necessary to pass threadId ? If yes, can you show me where it is used.